package.json

```diff
@@ -33,7 +33,7 @@
       "pre-commit": "lint-staged"
     }
   },
-  "packageManager": "pnpm@10.10.0",
+  "packageManager": "pnpm@10.15.0+sha512.486ebc259d3e999a4e8691ce03b5cac4a71cbeca39372a9b762cb500cfdf0873e2cb16abe3d951b1ee2cf012503f027b98b6584e4df22524e0c7450d9ec7aa7b",
```

Comment:
Is this specific version really needed?

Reply:
No, I just bumped it because of the warnings I was seeing on my console. If we do not want to update the patch, I'm happy to revert since I do not have any strong opinion on this.